src/OpenTelemetry.Instrumentation.AWS/OpenTelemetry.Instrumentation.AWS.csproj

```diff
@@ -9,9 +9,9 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AWSSDK.Core" Version="3.5.1.24" />
-    <PackageReference Include="AWSSDK.SimpleNotificationService" Version="3.5.0.27" />
-    <PackageReference Include="AWSSDK.SQS" Version="3.5.0.26" />
+    <PackageReference Include="AWSSDK.Core" Version="[3.5.1.24,4.0)" />
```

Comment:
Yes, sorrry, I've only just discovered ranges (as opposed to wildcards) in PackageReference. My intent is to allow the use of this package with apps that are using the current 3.7.x AWS packages without warnings about 3.5 dependencies. Whatever achieves that goal while allowing the broadest usage is fine by me.